code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -770,19 +790,636 @@ nobiliumsuppression = INFINITY
+	min_requirements = list(
+		/datum/gas/bz = MINIMUM_MOLE_COUNT,
+		/datum/gas/freon = MINIMUM_MOLE_COUNT,
+		"TEMP" = 50,
+		"MAX_TEMP" = 100
+	)
+
+/datum/gas_reaction/healium_formation/react(datum/gas_mixture/air, datum/holder)
+	var/list/cached_gases = air.gases
+	var/temperature = air.temperature
+	var/old_heat_capacity = air.heat_capacity()
+	var/heat_efficency = min(temperature * 0.3, cached_gases[/datum/gas/freon][MOLES], cached_gases[/datum/gas/bz][MOLES])
+	var/energy_used = heat_efficency * 100
+	ASSERT_GAS(/datum/gas/healium, air)
+	if ((cached_gases[/datum/gas/freon][MOLES] - heat_efficency * 2.75 < 0 ) || (cached_gases[/datum/gas/bz][MOLES] - heat_efficency * 0.25 < 0)) //Shouldn't produce gas from nothing.
+		return NO_REACTION
+	cached_gases[/datum/gas/freon][MOLES] -= heat_efficency * 2.75
+	cached_gases[/datum/gas/bz][MOLES] -= heat_efficency * 0.25
+	cached_gases[/datum/gas/healium][MOLES] += heat_efficency * 3
+
+	if(energy_used)
+		var/new_heat_capacity = air.heat_capacity()
+		if(new_heat_capacity > MINIMUM_HEAT_CAPACITY)
+			air.temperature = max(((temperature * old_heat_capacity + energy_used) / new_heat_capacity), TCMB)
+	return REACTING
+
+/datum/gas_reaction/proto_nitrate_formation
+	priority = 13
+	name = "Proto Nitrate formation"
+	id = "proto_nitrate_formation"
+
+/datum/gas_reaction/proto_nitrate_formation/init_reqs()
+	min_requirements = list(
+		/datum/gas/pluoxium = MINIMUM_MOLE_COUNT,
+		/datum/gas/hydrogen = MINIMUM_MOLE_COUNT,
+		"TEMP" = 5000,
+		"MAX_TEMP" = 10000
+	)
+
+/datum/gas_reaction/proto_nitrate_formation/react(datum/gas_mixture/air, datum/holder)
+	var/list/cached_gases = air.gases
+	var/temperature = air.temperature
+	var/old_heat_capacity = air.heat_capacity()
+	var/heat_efficency = min(temperature * 0.005, cached_gases[/datum/gas/pluoxium][MOLES], cached_gases[/datum/gas/hydrogen][MOLES])
+	var/energy_used = heat_efficency * 100
+	ASSERT_GAS(/datum/gas/proto_nitrate, air)
+	if ((cached_gases[/datum/gas/pluoxium][MOLES] - heat_efficency * 0.2 < 0 ) || (cached_gases[/datum/gas/hydrogen][MOLES] - heat_efficency * 2 < 0)) //Shouldn't produce gas from nothing.
+		return NO_REACTION
+	if(air.temperature >= 10000)
+		return NO_REACTION
+	cached_gases[/datum/gas/hydrogen][MOLES] -= heat_efficency * 2
+	cached_gases[/datum/gas/pluoxium][MOLES] -= heat_efficency * 0.2
+	cached_gases[/datum/gas/proto_nitrate][MOLES] += heat_efficency * 2.2
+
+	if(energy_used > 0)
+		var/new_heat_capacity = air.heat_capacity()
+		if(new_heat_capacity > MINIMUM_HEAT_CAPACITY)
+			air.temperature = max(((temperature * old_heat_capacity + energy_used) / new_heat_capacity), TCMB)
+	return REACTING
+
+/datum/gas_reaction/cyrion_b_formation
+	priority = 14
+	name = "Cyrion B formation"
+	id = "cyrion_b_formation"
+
+/datum/gas_reaction/cyrion_b_formation/init_reqs()
+	min_requirements = list(
+		/datum/gas/hypernoblium = MINIMUM_MOLE_COUNT,
+		/datum/gas/stimulum = MINIMUM_MOLE_COUNT,
+		"TEMP" = 260,
+		"MAX_TEMP" = 290
```

Comment:
You can make this a bit higher yes? If they have this they're prob making fusion already

Reply:
sure, since they have fusion how about 50000 K?